doc/retype.yml

```diff
@@ -0,0 +1,11 @@
+input: .
+output: .retype
+url: # Add your website address here
+branding:
+  title: Project Name
+  label: Docs
+links:
+- text: Getting Started
```

Comment:
Is it the actual title ? 
The link should be modified to point to docs.osmium-toolchains.xyz

Reply:
We do no have "Getting Started" section at the moment so I deleted it